.github/workflows/test.yaml

```
@@ -14,6 +14,16 @@ on:
         type: string
 
 jobs:
+  conda-cpp-checks:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-post-build-checks.yaml@branch-23.12
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
```

```suggestion
      build_type: nightly
```

`test.yaml` is only used for nightlies, so we can hard code this value here.